src/components/NewMessageForm/AudioRecorder/AudioRecorder.vue

```diff
@@ -277,7 +277,7 @@ export default {
 			const today = new Date()
 			let time = today.getFullYear() + '-' + ('0' + today.getMonth()).slice(-2) + '-' + ('0' + today.getDay()).slice(-2)
 			time += ' ' + ('0' + today.getHours()).slice(-2) + '-' + ('0' + today.getMinutes()).slice(-2) + '-' + ('0' + today.getSeconds()).slice(-2)
-			return t('spreed', 'Talk recording from {time} ({conversation})', { time, conversation }) + '.mp3'
+			return t('spreed', 'Talk recording from {time} ({conversation})', { time, conversation }) + '.m4a'
```

Comment:
Now it's not marked as a voice message anymore *roll eyes*

Reply:
Also since m4a is not an audio file format, safari doesn't allow playing it in the audio tag.